### DISCUSSION TITLE
missing jumpbox vm

### DISCUSSION THREAD

**Discussion by allwebcasts** [Azure/AKS-Construction#533] [Ideas]
I am new to AKS. As this is a private AKS cluster, I can not find a jumpbox VM. there is wel an option Azure bastion, but witout VM

**Comment by Gordonby** [comment]
AKS has a feature called `Run Command`. It negates the need for a jumpbox for most scenarios.
https://learn.microsoft.com/en-us/azure/aks/command-invoke

There is an option for Bastion in the AKSC Helper/Bicep, as several users of the project add a subnet for jumpboxes, but I don't know if it's a core scenario we'd want to tackle.

**Comment by allwebcasts** [comment]
Thanks for your quick reply. Jumbox is mostely used as you can install  your own tool, Helm, kubectl etc...

https://learn.microsoft.com/en-us/azure/architecture/example-scenario/aks-firewall/aks-firewall
https://github.com/Azure/AKS-Landing-Zone-Accelerator/tree/main/Scenarios/AKS-Secure-Baseline-PrivateCluster/Bicep/03-Network-Hub/modules/VM
https://github.com/Azure/azure-quickstart-templates/tree/master/demos/private-aks-cluster
https://denniszielke.medium.com/fully-private-aks-clusters-without-any-public-ips-finally-7f5688411184
